PictureApp.API/Data/SeedData/Seed.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using PictureApp.API.Models;
+
+namespace PictureApp.API.Data.SeedData
+{
+    public class Seed
+    {
+     private readonly DataContext context;
+        public Seed(DataContext context)
+        {
+            this.context = context;
+        }
+
+        public void SeedUsers()
+        {
+            var userData = System.IO.File.ReadAllText("Data/SeedData/20181119_UserSeedData.json");
+
+            var users = Newtonsoft.Json.JsonConvert.DeserializeObject<List<User>>(userData);
+
+            foreach (var user in users)
+            {
+                byte[] passwordHash, passwordSalt;
+                CreatePasswordHash("password", out passwordHash, out passwordSalt);
+
+                user.PasswordHash = passwordHash;
+                user.PasswordSalt = passwordSalt;
+                user.Username = user.Username.ToLower();
+
+                context.Users.Add(user);
+           }
+            context.SaveChanges();
+        }
+
+        private void CreatePasswordHash(string password, out byte[] passwordHash, out byte[] passwordSalt)
```

Comment:
Seed Data is just only for preparing temp user data based on json file (data/seedData/20181119_UserSeedData.json). Seed.cs is it independent fileused only to generate big part of user data. That file will be triggered occasionally

Reply:
Yes, I know what is the purpose of seed data and seed.cs file itself. However, it (this particular usage) may suggest to use single encapsulated algorithm for generating password hashes in different scenarios rather than copy & paste technique ;) Such approach meet two goals: better testability and consistency. No matter where it is used finally.

Reply:
te same situation, i removed the seed data from startup.cs file. commit
https://github.com/anowakowski/PictureApp_API/pull/26/commits/64b0406d0c66f72c8dbe0f44b13213efd384b5b9